test/mocks/MockPyth.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
why is this showing up as new? was this not here before?

Reply:
we had them only in oval demo repo